src/test/run-make-fulldeps/print-calling-conventions/Makefile

```diff
@@ -0,0 +1,4 @@
+-include ../tools.mk
+
+all:
+	$(RUSTC) --print calling-conventions
```

Comment:
Hm, wouldn’t this be better off as a UI test?

Reply:
Not sure - I'm following the example set by other similar tests. If we want to change the group of them to test in a different way, that seems like a separate change IMO https://github.com/rust-lang/rust/pull/100488#discussion_r945289589